source/manual/dns.html.md

```diff
@@ -45,39 +45,106 @@ The deployment process is complex, so it's much easier to [use the automation pr
 
 gds-cli provides a command which automates gathering the required credentials and triggering Jenkins. The command authenticates to Jenkins using GitHub, so it expects `GITHUB_USERNAME` / `GITHUB_TOKEN` environment variables to be set. It calls out to the `aws` and `gcloud` command line tools to fetch the other required credentials.
 
-You can get the relevant help for your version of gds-cli with:
+Follow these steps to deploy a change to the DNS for *.service.gov.uk:
 
-```
-$ gds govuk dns --help
-NAME:
-   gds govuk dns - deploy DNS via Jenkins
+1. Get a personal access token for GitHub by navigating to the [personal access token](https://github.com/settings/tokens) page then generate a new token with the default permissions. Set this as an environment variable, alongside your GitHub username:
 
-USAGE:
-   gds govuk dns [command options] [arguments...]
+    ```
+    export GITHUB_TOKEN=<token_from_github_ui>
+    export GITHUB_USERNAME=<your_github_username>
+    ```
 
-OPTIONS:
-...
-```
+1. Check you can access AWS (in production as a poweruser) using the following, if not, follow the steps in the [Get Started documentation](/manual/get-started.html#8-get-aws-access).
+
+    ```
+    gds aws govuk-production-poweruser aws s3 ls
```

Comment:
If this command is successfully, should I be able to see a list of jobs?
For example:

```
2020-01-27 11:52:59 aptly-sync
2022-12-20 08:15:29 aws-athena-query-results-172025368201-eu-west-1
2022-07-15 12:00:27 datagovuk-production-ckan-organogram
2020-06-24 03:03:40 datagovuk-production-ckan-static-data
2020-06-24 04:07:20 dns-state-bucket-production
2020-06-24 08:14:03 gds-public-readable-tarballs
2023-01-04 15:23:43 govuk-analytics-logs-production
.
.
.
```

Reply:
Yes, a list of S3 buckets.  Will add a sentence explaining what you should see.